src/main/java/com/gluonhq/substrate/util/ios/Deploy.java

```diff
@@ -104,9 +104,10 @@ private void checkPrerequisites() throws IOException, InterruptedException {
                 checkPrerequisites();
             }
         } else {
-            // Check for ios-deploy version installed (it should be 1.10+)
+            // Check for ios-deploy version installed (it should be 1.11+)
             String version = ProcessRunner.runProcessForSingleOutput("ios-deploy version","ios-deploy", "-V");
-            if (version != null && !version.isEmpty() && (version.startsWith("1.8") || version.startsWith("1.9"))) {
+            if (version != null && !version.isEmpty() &&
+                    (version.startsWith("1.8") || version.startsWith("1.9") || version.startsWith("1.10"))) {
                 Logger.logDebug("ios-deploy version (" + version + ") is outdated");
```

Comment:
What should developers do when they see this?

Reply:
Nothing, the plugin will uninstall the old version and install the new one:
```
Logger.logDebug("ios-deploy version (" + version + ") is outdated");
uninstallIOSDeploy();
if (installIOSDeploy()) {
    checkPrerequisites();
}
```

Reply:
Perhaps the log message should mention that substrate will try to automatically update to the required version?

Reply:
Fixed